docs/source/en/quicktour.mdx

```diff
@@ -100,13 +74,7 @@ label: POSITIVE, with score: 0.9998
 label: NEGATIVE, with score: 0.5309
 ```
 
-The [`pipeline`] can also iterate over an entire dataset. Start by installing the [🤗 Datasets](https://huggingface.co/docs/datasets/) library:
-
-```bash
-pip install datasets 
-```
-
-Create a [`pipeline`] with the task you want to solve for and the model you want to use.
+The [`pipeline`] can also iterate over an entire dataset for a task like automatic speech recognition:
```

Comment:
Again, this gives the impression that this is specific to automatic speech recognition when it's general.